pyproject.toml

```diff
@@ -43,7 +43,7 @@ ths_ds_filter = 'toshi_hazard_store.scripts.ths_ds_filter:main'
 # ths_testing = 'toshi_hazard_store.scripts.ths_testing:cli'
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.13"
```

Comment:
some features of THS depend on openquake (unpacking OQ hdf5 for realization storage). OQ doesn't officially support py3.13 yet. I don't know if this will cause any issues, but be aware of it